docs/index.rst

```diff
@@ -36,6 +36,11 @@ User Guide
    user/buildroot_overrides
    user/man_pages/index
 
+.. toctree::
+   :maxdepth: 1
```

Comment:
No need to duplicate toctree and maxdepth. If you don't want to expand subchapters, you can move  `user/fedora-flavored-markdown` just above `user/release_notes`, otherwise you can put the line between `user/buildroot_overrides` and `user/man_pages/index`

Reply:
Sounds good!